homeassistant/components/sensor/bom_weather_current.py

```diff
@@ -0,0 +1,188 @@
+"""
+Support for bom.gov.au current condition weather service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.bom_weather_current
+"""
+
+import logging
+import requests
+import voluptuous as vol
+import datetime
+
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.const import (
+    CONF_MONITORED_CONDITIONS, TEMP_CELSIUS,
+    STATE_UNKNOWN, CONF_NAME)
+
+_RESOURCE = 'http://www.bom.gov.au/fwo/{}/{}.{}.json'
+_LOGGER = logging.getLogger(__name__)
+
+CONF_ZONE_ID = 'zone_id'
+CONF_WMO_ID = 'wmo_id'
+
+MIN_TIME_BETWEEN_UPDATES = datetime.timedelta(seconds=60)
```

Comment:
We are talking about weather data. Usually it doesn't matter if there is a delay. It's your call and it looks that you spent some thoughs on this.